packages/expo-enclave/example/package.json

```diff
@@ -9,6 +9,8 @@
     "web": "expo start --web"
   },
   "dependencies": {
+    "daimo-contract-types": "file:daimo-contract-types-1.0.0.tgz",
+    "daimo-userop": "file:daimo-userop-1.0.0.tgz",
```

Comment:
using tgz here is very weird, apparently react native is unable to take as input local dependencies (stuff with `file:..`), see https://github.com/facebook/react-native/issues/29977 and this seems like one of the suggested fixes. There's probably a better fix by using workspaces or something as in one of the comments above, todo here to investigate/fix